src/njoy21/input/RESXSR/Card5/Mat.hpp

```diff
@@ -0,0 +1,13 @@
+struct Mat {
+  using Value_t = unsigned int;
```

Comment:
@apmccartney is very nervous about `unsigned int`. Ask him about it some time (if you can catch him).